README.md

```diff
@@ -1,150 +1,16 @@
-to run designs through the flow
-
-## Useful Resources
-Your feedback is very welcome.
-- **Website**: Visit our [website](https://theopenroadproject.org/) for more
-  information on the project, participants and objectives.
-- **GitHub Issues**: Please submit GitHub issues for any bugs, improvements, or
-  new feature requests. Feedback can be directed at this repository or the
-  repository for the tool, component or design as appropriate. This
-  [link](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+archived%3Afalse+user%3AThe-OpenROAD-Project+)
-  is an aggregate of all OpenROAD related issues
-- **Community**: Find us on the public [Gitter
-  community](https://gitter.im/The-OpenROAD-Project/community) where you can ask
-  quick questions or discuss anything OpenROAD related.
-- **Twitter**: Follow us on twitter -
-  [@OpenROAD_EDA](https://twitter.com/OpenROAD_EDA).
 
-## Contributing
-We welcome any comments, patches and designs to help us improve the tool. At
-this time we are focused on overhauling the build process, build testing, and
-the continuous integration framework. This will set us up for better integration
-with contributors. Please stay tuned.
-
-## About the Project
-The OpenROAD ("Foundations and Realization of Open, Accessible Design") project
-was launched in June 2018 within the DARPA IDEA program. OpenROAD aims to bring
-down the barriers of cost, expertise and unpredictability that currently block
-designers' access to hardware implementation in advanced technologies. The
-project team (Qualcomm, Arm and multiple universities and partners, led by UC
-San Diego) is developing a fully autonomous, open-source tool chain for digital
-layout generation across die, package and board, with initial focus on the
-RTL-to-GDSII phase of system-on-chip design. Thus, OpenROAD holistically attacks
-the multiple facets of today's design cost crisis: engineering resources, design
-tool licenses, project schedule, and risk.
-
-The [IDEA program](https://www.darpa.mil/program/intelligent-design-of-electronic-assets)
-targets no-human-in-loop (NHIL) design, with 24-hour turnaround time and eventual
-zero loss of power-performance-area (PPA) design quality. No humans means that
-tools must adapt and self-tune, and never get stuck: thus, machine intelligence
-must replace today's human intelligence within the layout generation process. 24
-hours means that problems must be aggressively decomposed into bite-sized
-subproblems for the design process to remain within the schedule constraint.
-Eventual zero loss of PPA quality requires parallel and distributed search to
-recoup the solution quality lost by problem decomposition.
+See the OpenROAD [documentation here](https://openroad.readthedocs.io/en/latest/).
+How to [start using OpenROAD flow here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html)
+and our [user guide here](https://openroad.readthedocs.io/en/latest/user/UserGuide.html).
```

Comment:
Can this be a bullet list? It would be nice to have each of these items on a separate line.

Reply:
Fixed on 697aad5